packages/next/src/server/base-http/web.ts

```diff
@@ -113,10 +110,11 @@ export class WebNextResponse extends BaseNextResponse<WritableStream> {
 
     let bodyInit: BodyInit = body
 
+    // if the response is streaming, onClose() still be called after this point.
```

Comment:
"_will_ be called after this point"? or "_may still_ be called after this point"? I'm actually not sure.